client/src/i18n/fr/stock.json

```diff
@@ -142,10 +142,6 @@
       "AVERAGE_MONTHLY_CONSUMPTION"  : "Consommation Mensuelle Moyenne",
       "ALGO_1"          : "Algorithme 1",
       "ALGO_1_COMMENT"  : "La consommation moyenne mensuel est obtenue en divisant la quantité consommée durant la période définie par le nombre de jour de stock pour ladite période, et en multipliant le résultat obtenu par 30.5",
-      "ALGO_2"          : "Algorithme 2",
-      "ALGO_2_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre de jours de consommation pour ladite période, et en multipliant le résultat obtenu par 30.5",
-      "ALGO_3"          : "Algorithme 3",
-      "ALGO_3_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre des jours de la période, et en multipliant le résultat obtenu par 30.5",
       "ALGO_4"          : "Algorithme 4 (MSH)",
       "ALGO_4_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre de mois moins le nombre de mois divisé par le total de nombre de jour de rupture",
```

Comment:
L'explication de l'algorithme MSH est un peu difficile à lire actuellement, que pensez-vous de : 
```
La consommation moyenne est égale à : 

La quantité consommée durant la période / (Nombre de mois à considérer - (Le nombre de jour de rupture de stock / 30.5)). 

L'algorithme MSH est recommandé par l'organisation Management Sciences for Health (https://www.msh.org).
```

Reply:
Thanks!  This is a great change.  I'll make it.  👍